test/e2e-common.sh

```diff
@@ -571,12 +574,13 @@ function stage_test_resources() {
       continue
     fi
     target="${file/${source_dir}/$target_dir}"
-    mkdir -p $(dirname $target)
+    mkdir -p "$(dirname "${target}")"
 
     if grep -Fq "ko://" "${file}"; then
-      local ko_target="$(mktemp -d)/$(basename $file)"
+      local ko_target
+      ko_target="$(mktemp -d)/$(basename "${file}")"
       echo building "${file/$REPO_ROOT_DIR/}"
-      ko resolve $(ko_flags) -f "${file}" > "${ko_target}" || fail_test "failed to build test resource"
+      ko resolve "$(ko_flags)" -f "${file}" > "${ko_target}" || fail_test "failed to build test resource"
```

Comment:
this can't be quoted

- see the errors in the prow job

Reply:
Fixed